test/e2e/framework/util.go

```diff
@@ -476,3 +488,66 @@ func (f *Framework) GetNodeName(meta metav1.ObjectMeta) string {
 	// if none of above succeed, return default testing node "minikube"
 	return "minikube"
 }
+
+func (f *Framework) CreateSampleDataInsideWorkload(meta metav1.ObjectMeta, resourceKind string) error {
+	switch resourceKind {
+	case apis.KindDeployment, apis.KindReplicaSet, apis.KindReplicationController:
+		pod, err := f.GetPod(meta)
+		if err != nil {
+			return err
+		}
+		file := "test-data.txt"
+		_, err = f.ExecOnPod(pod, "touch", filepath.Join(TestSourceDataMountPath, file))
+		if err != nil {
+			return err
+		}
+	case apis.KindStatefulSet, apis.KindDaemonSet:
+		pods, err := f.GetAllPod(meta)
+		if err != nil {
+			return err
+		}
+		files := []string{"test-data1.txt", "test-data2.txt", "test-data3.txt", "test-data4.txt", "test-data5.txt"}
+		for i, pod := range pods {
+			_, err := f.ExecOnPod(&pod, "mkdir", "-p", "/source/data/dir-"+strconv.Itoa(i))
```

Comment:
don't use `/source/data/dir`. use `filepath.Join("source","data","dir-"+strconv.Itoa(i))`. otherwise it won't work on Windows os.